src/sonic_pc/data_structures.rs

```diff
@@ -8,7 +8,7 @@ use ark_ec::{PairingEngine, ProjectiveCurve};
 pub type UniversalParams<E> = kzg10::UniversalParams<E>;
 
 /// `Randomness` is the randomness for the KZG10 scheme.
-pub type Randomness<E> = kzg10::Randomness<E>;
+pub type Randomness<E, P> = kzg10::Randomness<E, P>;
```

Comment:
Do we want to give the same treatment here, or should we reserve that for a future PR?